funcao-divisao/funcaoDivisao.js

```diff
@@ -0,0 +1,11 @@
+function divisao(x, y) {
+  let z = x / y;
```

Comment:
O que acontece se seu y for um número negativo?

Reply:
z é negativo, se for 0 infinty